### ISSUE TITLE
rustc --explain E0487 is inconsistent and unclear

### ISSUE THREAD

**Issue by ryanswilson59** [rust-lang/rust#67040]
In the text visible at https://doc.rust-lang.org/1.25.0/error-index.html#E0478
We first see 

> In this example, the 'SnowWhite lifetime is supposed to outlive the 'kiss lifetime 

```
struct Prince<'kiss, 'SnowWhite: 'kiss> { // You say here that 'kiss must live
                                          // longer than 'SnowWhite.
    child: Box<Wedding<'kiss> + 'SnowWhite>, // And now it's all good!
}
```

The comment directly contradicts that 'SnowWhite should outlive 'kiss, thus making it unclear what the code actually does.

**Comment by JohnTitor** [comment]
@ryanswilson59 This has been fixed by #67168?

**Comment by JohnTitor** [comment]
Closing as resolved, feel free to re-open if it's still an issue.
